src/js/component.js

```diff
@@ -1398,14 +1399,26 @@ class Component {
    * @method getComponent
    */
   static getComponent(name) {
-    if (Component.components_ && Component.components_[name]) {
-      return Component.components_[name];
+    if (!name) {
```

Comment:
will update, i don't know the state of the formatting rules, will ignore http://eslint.org/docs/rules/no-param-reassign.